packages/react-scripts/scripts/build.js

```diff
@@ -128,6 +128,13 @@ function build(previousSizeMap) {
       process.exit(1);
     }
 
+    // Webpack may swallow uglify errors even with bail:true
+    if (stats.compilation.errors.length) {
+      console.error(chalk.red('Error(s) occurred during the production build:'));
```

Comment:
Can this be the same message that we use for failed compilation in `start.js`? And would be nice to fix it up for the other (different) message in this file so they are all consistent.